dyn_em/solve_em.F

```diff
@@ -3754,6 +3765,23 @@ SUBROUTINE solve_em ( grid , config_flags  &
 !       &        , QZH_CURR=scalar(ims,kms,jms,P_QZH), F_QZH=F_QZH          & ! "
       &        , QVOLG_CURR=scalar(ims,kms,jms,P_QVOLG), F_QVOLG=F_QVOLG    & ! for nssl_2mom
       &        , QVOLH_CURR=scalar(ims,kms,jms,P_QVOLH), F_QVOLH=F_QVOLH    & ! for nssl_2mom
+!----------------------------------for ntu3m--------------------------------------------------
```

Comment:
@tzuchin12 
Typically, developers denote their variables with a comment after the "&" character, instead of 
```
!----------------------------------for ntu3m----
```

Reply:
Modified as suggested.